scripts/test/launchTestUtilities.ts

```diff
@@ -75,45 +201,9 @@ export async function invokeVSCodeCli(vsCodeExecutablePath: string, args: string
     return spawnResult.stdout
 }
 
-export async function installVSCodeExtension(vsCodeExecutablePath: string, extensionIdentifier: string): Promise<void> {
-    // HACK: `sam.test.ts` Codelens test was failing for python due to bug in newer version, so lock to last working version.
-    // Edge Case: This specific python version does not work with the "minimum" vscode version, so we do not override it as it
-    // will choose its own python extension version that works.
-    if (extensionIdentifier === VSCODE_EXTENSION_ID.python && process.env[envvarVscodeTestVersion] !== minimum) {
-        extensionIdentifier = `${VSCODE_EXTENSION_ID.python}@2023.20.0`
-    }
-
-    console.log(`Installing VS Code Extension: ${extensionIdentifier}`)
-    console.log(await invokeVSCodeCli(vsCodeExecutablePath, ['--install-extension', extensionIdentifier]))
-}
-
 /**
- * Alternative to the `--disable-extensions` CLI flag which allows us to
- * selectively keep some extensions enabled.
- *
- * @param vsCodeExecutablePath Path to vscode CLI program.
- * @param exceptIds List of extension ids that should *not* be disabled.
- *
- * @returns List of args which the caller is expected to pass to vscode CLI, of the form:
- * ["--disable-extension", "foo.bar.baz", "--disable-extension", ...]
+ * TODO: Look to not export this, but a test needs it currently
```

Comment:
the tests should use this instead: https://github.com/aws/aws-toolkit-vscode/blob/02f5ef9f8ee2ac4669c27e5d73d22abc446c18ae/src/shared/vscode/env.ts#L69

and maybe we can eliminate this one.